Cargo.toml

```diff
@@ -135,4 +135,5 @@ members = [
 	"chainspec",
 	"ethcore/wasm/run",
 	"evmbin",
+	"util/EIP-2124"
```

Comment:
The only reason crates get listed here is if they provide binaries that need to be built as part of the whole, like `evmbin`, `chainspec` etc.

Other crates in the repo are dependencies of one or more of the above so they don't need to appear here.

(It's a perfectly fine thing to do while working on it ofc)

Reply:
I agree and it is only listed here to make Cargo work before it is included in `parity` dependency tree.

Reply:
this can be removed when it's used in the main binary